pkg/translator/azure/resources_to_traces.go

```diff
@@ -62,8 +63,10 @@ type azureTracesRecord struct {
 var _ ptrace.Unmarshaler = (*TracesUnmarshaler)(nil)
 
 type TracesUnmarshaler struct {
-	Version string
-	Logger  *zap.Logger
+	Version    string
+	Logger     *zap.Logger
+	TimeFormat []string
+	TimeOffset time.Duration
```

Comment:
Please add tests to cover this path as well

Reply:
removed `time_offset` and added more cases for `time_format`.